### PR TITLE
solved orderSummary issue in different pages

### LINKED ISSUE #1893: Bug : Fixing some bugs
Screenshot (2174)
Screenshot (2175)
Screenshot (2173)

### PR DESCRIPTION
**Pull request by SadafKausar2025**
## Fixes Issue #1893

Closes #1893

## Screenshots
Screenshot (2173)

## Summary by CodeRabbit

- **New Features**
  - Updated Order Summary component with refined structure and presentation.
  - Integrated SweetAlert2 for improved user feedback in the Cart page.
  - Enhanced logout experience with a confirmation dialog in the User Navbar.
  - Improved Product Card functionality with updated button label and state management.

- **Style**
  - Enhanced styling and layout consistency in the Checkout and Cart pages.
  - Adjusted styling and structure in the Order Summary component for a better user experience, including improved visibility of the subtotal and shipping cost.
  - Improved responsiveness and layout of navigation links and dropdowns in the User Navbar.

### CODE REVIEW

File: src/User/components/Order/OrderSummary.jsx
Comment (codervivek5) [inline src/User/components/Order/OrderSummary.jsx:1]: chech this file....payment is properly working or not

File: src/User/components/Navbar/UserNavbar.jsx [hunk 1]
```diff
@@ -6,80 +6,149 @@ import CartIcon from "./CartIcon";
 import AuthButton from "./AuthButton";
 import MobileMenu from "./MobileMenu";
 import { FaUserCircle } from "react-icons/fa";
+import Swal from "sweetalert2";
+import "../../components/FeedbackForm/Sweetpopup.css";
 
-const UserNavbar = ({ isAdmin }) => {
+const Navbar = ({ isAdmin }) => {
```
Comment (codervivek5) [inline src/User/components/Navbar/UserNavbar.jsx]: check this line

File: src/User/components/Navbar/UserNavbar.jsx [hunk 2]
```diff
@@ -148,4 +227,4 @@ const UserNavbar = ({ isAdmin }) => {
   );
 };
 
-export default UserNavbar;
\ No newline at end of file
+export default Navbar;
```
Comment (codervivek5) [inline src/User/components/Navbar/UserNavbar.jsx]: also check this

File: src/User/components/Order/OrderSummary.jsx
Comment (codervivek5) [inline src/User/components/Order/OrderSummary.jsx:1]: do not make any change in this file

**Review by codervivek5 — CHANGES_REQUESTED** [review]

File: src/User/components/Order/OrderSummary.jsx
Comment (SadafKausar2025) [reply src/User/components/Order/OrderSummary.jsx:1]: done

File: src/User/pages/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -192,6 +190,7 @@ const Dashboard = () => {
                 title={product.title}
                 price={product.price}
                 rating={product.rating}
+                product={product}
```
Comment (codervivek5) [inline src/User/pages/Dashboard/Dashboard.jsx]: check this one

File: src/User/pages/Order/Cart.jsx
Comment (codervivek5) [inline src/User/pages/Order/Cart.jsx:1]: check sweet alert issue

**Review by codervivek5 — CHANGES_REQUESTED** [review]

**Pull request merged**